dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/validation/ValidationController.java

```diff
@@ -107,7 +108,7 @@ public ValidationSummary validate(
   @RequestMapping(
       value = "/sendNotifications",
       method = {RequestMethod.PUT, RequestMethod.POST})
-  @PreAuthorize("hasRole('ALL') or hasRole('M_dhis-web-app-management')")
+  @RequiresAuthority(anyOf = Authorities.M_DHIS_WEB_APP_MANAGEMENT)
```

Comment:
Is this a potential bug?
Should this be `F_RUN_VALIDATION`?

Reply:
:thinking: good catch - it feels off - needs investigation to see if we can restore the knowledge of the intent here